clang/runtime/dpct-rt/include/dpl_extras/iterators.h.inc

```diff
@@ -179,6 +179,8 @@ constant_iterator<_Tp> make_constant_iterator(_Tp __value) {
 // dereferenced arg_index_input_iterator
 template <typename _KeyTp, typename _ValueTp> class key_value_pair {
 public:
+  key_value_pair() = default;
```

Comment:
Should we add std::enable_if to make sure _KeyTp and _ValueTp have default ctor?